Q1/imyogeshgaur/Solution.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+using namespace std;
+bool PallinDrome(int num)
+{
+    int div = 1;
+    while (num / div >= 10)
+        div *= 10;
+    while (num != 0)
+    {
+        int first = num / div;
+        int last = num % 10;
+        if (first != last)
+            return false;
+        num = (num % div) / 10;
+        div = div / 100;
+    }
+    return true;
+}
+bool isPallinDrome(int sum)
+{
+    if (PallinDrome(sum))
+        return true;
+    else
+        return false;
+}
+int main()
+{
+    int N, sum = 0, div = 1;
+    cout << "Enter the Number : ";
+    cin >> sum;
```

Comment:
Are you sure about this that the input should be sum? @imyogeshgaur

Reply:
I think i have added it by mistake and not test it afterward but i will correct it 😊

Reply:
thanks for the review 👍🏻 @Yatharth0045